src/services/tokenService.js

```diff
@@ -20,3 +21,52 @@ export default async function getToken(
 
   return token;
 }
+
+
+
+export async function getUserToken(name) {
+
+  const extractUrlCode = () => {
```

Comment:
it is better to get the code in the component and pass it to this function I think.

Reply:
this function is also being passed from custom-app
so we have to make change ther also for this